jpegenc/subblocks/RLE/RLECore/entropycoder.py

```diff
@@ -8,31 +8,31 @@ def two2bin(num):
     return inum + 1
 
 
-def nbits(num, width):
+def nbits(num, width_data):
```

Comment:
I don't understand this function?  What does this do different than `int.bit_length()` or `log(num, 2)`?   
(there might be something I just am not currently understanding?)